tools/validate-deployment.sh

```diff
@@ -60,7 +62,7 @@ if [[ $ipaddress =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
         echo "[ERROR] Public key missing when resetting key for ProctorVM for $teamAAD - Subscription $subid - Exiting ..."
     fi
 
```

Comment:
kubeconfig file does not have the proper permissions hence you need to chown it.
`             ssh -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE azureuser@$ipaddress "bash -s" << EOF
                    sudo chown azureuser:azureuser /home/nginx/contents/kubeconfig;
EOF
`